scripts/dockerfile-assets/nginx/default.conf.template

```diff
@@ -14,11 +14,11 @@ server {
         add_header Cache-Control "no-store, no-cache, must-revalidate";
     }
 
-    location ~ "^$BASEPATH(.+\.[0-9a-f]{20}\..+)$" {
+    location ~ "^$BASEPATH(.+\.[0-9a-f]{16,20}\..+)$" {
```

Comment:
```suggestion
    location ~ "^$BASEPATH(.+\.([0-9a-f]{16}|[0-9a-f]{20})\..+)$" {
```

Would it not then make sense to match either 16 or 20 times instead of taking everything in between as well?

Reply:
Agreed, though I think more specific is almost always better this should likely be fine.